roles/fs-drift/templates/workload_job.yml.j2

```diff
@@ -42,29 +42,28 @@ spec:
           - name: uuid
             value: "{{ uuid }}"
           - name: test_user
-            value: "{{ test_user }}"
+            value: "{{ test_user | default("ripsaw") }}"
           - name: clustername
             value: "{{ clustername }}"
-{% if elasticsearch.server is defined %}
           - name: es
-            value: "{{ elasticsearch.server }}"
+            value: "{{ elasticsearch.server | default() }}"
           - name: es_port
-            value: "{{ elasticsearch.port }}"
+            value: "{{ elasticsearch.port | default() }}"
           - name: es_index
```

Comment:
shouldn't this default as well?   You know you're in ripsaw at this point so why not default to ripsaw-fs-drift ?

Reply:
I don't understand. It's already set to "ripsaw-fs-drift"

Reply:
this default could be computed from "ripsaw-{{ spec.workload.name }}" once, by ripsaw, not each benchmark.

Reply:
We can't do that now since that the workload name doesn't mach with the index name, we would have rename the workloads to do so. I think that should be done in a different PR.

Reply:
didn't know that, ok then I'll withdraw that suggestion.